aws_cloudtrail_policies/aws_cloudtrail_s3_bucket_public.py

```diff
@@ -10,7 +12,7 @@ def policy(resource):
     bucket = resource_lookup(bucket_arn)
 
     for grant in bucket['Grants'] or []:
```

Comment:
```suggestion
    for grant in bucket.get('Grants', []):
```

Reply:
Unfortunately this one doesn't work when the `Grants` key is present, but has a value of `None`. That's because `bucket.get('Grants, [])` will only return `[]` if the `Grants` key doesn't exist. In this case, the key does exist but is mapped to `None`. I think for now we need to stick with how it is here.